NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.3 2021-12-15
```

Comment:
Please remove the date (replace with IN-PROGRESS or similar) as it is incorrect until the module is released